test/services/licences/fetch-agreements.service.test.js

```diff
@@ -8,46 +8,62 @@ const { describe, it, beforeEach } = exports.lab = Lab.script()
 const { expect } = Code
 
 // Test helpers
-const DatabaseSupport = require('../../support/database.js')
 const FinancialAgreementHelper = require('../../support/helpers/financial-agreement.helper.js')
 const LicenceAgreementHelper = require('../../support/helpers/licence-agreement.helper.js')
 
 // Thing under test
-const FetchAgreementsService =
-  require('../../../app/services/licences/fetch-agreements.service.js')
+const FetchAgreementsService = require('../../../app/services/licences/fetch-agreements.service.js')
 
 describe('Fetch Agreements service', () => {
-  const licenceRef = '01/12/34/1000'
-
-  beforeEach(async () => {
-    await DatabaseSupport.clean()
-  })
+  let licenceAgreement
+  let financialAgreement
 
   describe('when the licence has agreements data', () => {
     beforeEach(async () => {
-      const financialAgreement = await FinancialAgreementHelper.add({
-        id: '970168ce-06c3-4823-b84d-9da30b742bb8',
-        code: 'S127'
+      financialAgreement = await FinancialAgreementHelper.add()
+    })
+
+    describe('and the agreement has not been deleted', () => {
+      beforeEach(async () => {
+        licenceAgreement = await LicenceAgreementHelper.add({
+          endDate: new Date('2040-05-01'),
```

Comment:
`endDate`, `startDate` & `signedOn` are reused in all the tests. You could declare these all as consts above the lets on line 18.  You can then update your code where these are used so that it looks more like this:
`        licenceAgreement = await LicenceAgreementHelper.add({
          endDate,
          financialAgreementId: financialAgreement.id,
          startDate,
          signedOn
        })`